src/zmq/zmqabstractnotifier.cpp

```diff
@@ -20,3 +20,8 @@ bool CZMQAbstractNotifier::NotifyTransaction(const CTransaction &/*transaction*/
 {
     return true;
 }
+
+bool CZMQAbstractNotifier::NotifyTransactionEviction(const CTransaction &/*transaction*/, MemPoolRemovalReason &reason)
```

Comment:
I don't think it makes sense to pass `reason` as a reference here. Let alone a mutable one!

Reply:
Done.